### PR TITLE
Feature/responsive/add media queries

### PR DESCRIPTION
**Pull request by just-s-i-d**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -541,302 +570,527 @@ footer {
...
+        font-size: 10px;
+    }
+
+    .heading {
+        font-size: 32px;
+    }
+
+    .contact-section {
+        font-size: 10px;
+    }
```
Comment (shelly19gkmit) [inline css/style.css:751]: Can create a common class for both -
.testimonial-container {
        font-size: 10px;
    }
.contact-section {
        font-size: 10px;
    }

File: css/style.css [hunk 2]
```diff
@@ -541,302 +570,527 @@ footer {
...
+        display: flex;
+        flex-direction: column;
+        max-width: 100%;
+    }
+
+    .contact-form-container form input,
+    textarea {
+        margin: 8px 0;
+        padding: 3px;
+
```
Comment (shelly19gkmit) [inline css/style.css:767]: remove extra line

File: css/style.css [hunk 3]
```diff
@@ -541,302 +570,527 @@ footer {
...
     }
 
-    .brand-tagline-container .brand {
-        font-size: 3.8em;
-        letter-spacing: 5px;
+    .contact-form-container form input,
+    textarea {
+        margin: 8px 0;
+        padding: 3px;
+
```
Comment (shelly19gkmit) [inline css/style.css:859]: remove extra lines wherever present in this PR

File: css/style.css [hunk 4]
```diff
@@ -541,302 +570,527 @@ footer {
...
+    .navbar-container {
+        font-size: 20px;
     }
 
-    .brand-tagline-container .tagline {
-        font-size: 1.2em;
+    .brand-tagline-container {
+        width: 45%;
+        height: 50%;
+
```
Comment (shelly19gkmit) [inline css/style.css:1056]: same as above comment

Comment (just-s-i-d) [reply css/style.css:1056]: ok i'll work on them

**Pull request closed without merging**